download-references.nf

```diff
@@ -134,15 +140,21 @@ process download_star_fusion {
 
     script:
     """
-    aws s3 --no-sign-request --region ${params.awsregion} cp s3://ngi-igenomes/Homo_sapiens/Ensembl/GRCh38/Genome/CTAT/ctat_star_fusion_1_8_1.tar.gz .
-    tar -xf ctat_star_fusion_1_8_1.tar.gz --strip-components=5
-    rm ctat_star_fusion_1_8_1.tar.gz
+    if [[ ${params.genome} == "GRCh37" ]]; then
+      wget -O GRCh37_gencode_v19_CTAT_lib_Apr032020.plug-n-play.tar.gz https://data.broadinstitute.org/Trinity/CTAT_RESOURCE_LIB/__genome_libs_StarFv1.9/GRCh37_gencode_v19_CTAT_lib_Apr032020.plug-n-play.tar.gz
+      tar -xf GRCh37_gencode_v19_CTAT_lib_Apr032020.plug-n-play.tar.gz --strip-components=5
+      rm GRCh37_gencode_v19_CTAT_lib_Apr032020.plug-n-play.tar.gz
+    else
+      aws s3 --no-sign-request --region ${params.awsregion} cp s3://ngi-igenomes/Homo_sapiens/Ensembl/GRCh38/Genome/CTAT/ctat_star_fusion_1_8_1.tar.gz .
+      tar -xf ctat_star_fusion_1_8_1.tar.gz --strip-components=5
+      rm ctat_star_fusion_1_8_1.tar.gz
+    fi
```

Comment:
STAR-Fusion is a tricky tool to integrate. By default they use NCBI genome reference which we try to avoid. The reason is that most other tools are based on some version of Ensembl and I would to keep it that way. I am not sure if we should make an exception here. We have a build script in `build-ctat.nf`.
What do you think @MaxUlysse?

Reply:
Will revisit this and see how we can go with GRCh37 + Ensembl (which I also favour).

Reply:
I have edited `build-ctat.nf` with the previous version of `CTAT_HumanFusionLib` being used. The version you use `v0.2.0` is referred to [here](https://github.com/FusionAnnotator/CTAT_HumanFusionLib/releases) as being 'based on `GRCh38`', so I presume `v0.1.0` is `GRCh37`. I can ask in `issues` to confirm if you feel it is necessary?

Then we can specify in `usage.md` that to use `GRCh37` it is required to run `build-ctat.nf`?

Would this be OK?

Reply:
OK, have pushed those changes, tested `build-ctat.nf` and that is working and replaces previous `ctat_genome_lib_build_dir`. I also tested this with `main.nf` running `--star-fusion` and it completes without errors so I'm happy with that.